entrypoint.sh

```diff
@@ -29,7 +29,7 @@ do
 done
 
 echo "Running rake commands to initialize db and run migrations"
-rails db:create db:migrate
+rails db:create db:migrate db:seed
```

Comment:
This makes sense.  I suppose we could also replace this line with `rails db:setup`, which effectively does all 3 of these, but this makes sense.  Was another solution you were considering using sql files instead of doing the commands in seed.rb?